test/integration/index-index/test/index.test.ts

```diff
@@ -54,7 +57,8 @@ function runTests() {
     expect($('#page').text()).toBe('index > index')
   })
 
-  it('should client render page /index', async () => {
+  // FIXME: pages named "index" are never hydrated in Webpack during development
+  testNotWebpackDev('should client render page /index', async () => {
```

Comment:
@timneutkens These are the current tests highlighting the breakage in Pages Router with Webpack when pages are named `index`. They only worked because we just asserted on DOM attachment not visibility.